protoc_plugin/lib/src/code_generator.dart

```diff
@@ -96,7 +96,14 @@ class CodeGenerator {
             BytesBuilder(), (BytesBuilder builder, data) => builder..add(data))
         .then((builder) => builder.takeBytes())
         .then((List<int> bytes) {
-      var request = CodeGeneratorRequest.fromBuffer(bytes, extensions);
```

Comment:
This is normal protobuf API to decode messages. Do we want all users of this API to hit exceptions in production and then force them to rewrite their code like this?

It seems to me this 64 MB limit is completely arbitrary and seems to come from the very first version of this library in d6b19235.

I think it would make more sense to just remove this by instead changing it to:
```diff
diff --git a/protobuf/lib/src/protobuf/coded_buffer_reader.dart b/protobuf/lib/src/protobuf/coded_buffer_reader.dart
index 5333707..61bf401 100644
--- a/protobuf/lib/src/protobuf/coded_buffer_reader.dart
+++ b/protobuf/lib/src/protobuf/coded_buffer_reader.dart
@@ -9,8 +9,6 @@ part of protobuf;
 class CodedBufferReader {
   // ignore: constant_identifier_names
   static const int DEFAULT_RECURSION_LIMIT = 64;
-  // ignore: constant_identifier_names
-  static const int DEFAULT_SIZE_LIMIT = 64 << 20;
 
   final Uint8List _buffer;
   int _bufferPos = 0;
@@ -21,11 +19,12 @@ class CodedBufferReader {
   final int _sizeLimit;
 
   CodedBufferReader(List<int> buffer,
-      {int recursionLimit = DEFAULT_RECURSION_LIMIT,
-      int sizeLimit = DEFAULT_SIZE_LIMIT})
+      {int recursionLimit = DEFAULT_RECURSION_LIMIT, int sizeLimit = -1})
       : _buffer = buffer is Uint8List ? buffer : Uint8List.fromList(buffer),
         _recursionLimit = recursionLimit,
-        _sizeLimit = math.min(sizeLimit, buffer.length) {
+        _sizeLimit = sizeLimit >= 0
+            ? math.min(sizeLimit, buffer.length)
+            : buffer.length {
     _currentLimit = _sizeLimit;
   }
```

Reply:
We have #386 about `DEFAULT_SIZE_LIMIT`.